en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/adapter-configurations.md

```diff
@@ -396,7 +396,7 @@ See the example .toml file given below.
                     <div class="mb-config-example">
 <pre><code class="toml">[adapter.consul]
   enable = false
-  url = "https://169.254.1.1:8501" # scheme + host ip + port
+  URL = "https://169.254.1.1:8501" # scheme + host ip + port
```

Comment:
It is incorrect to have URL in upper case here.

Reply:
Good catch. Change reverted.